client/src/components/Buttons.js

```diff
@@ -80,3 +80,9 @@ export const ExitInterviewButton = (props) => {
   const { text } = props
   return <Button variant="outlined" className={classes.exitInterviewButton}>{text}</Button>
 }
+
+export const AnswerButton = (props) => {
+  const classes = useStyles()
+  const { text } = props
```

Comment:
Don't forget semicolons - are you using a linted?
Are we using this button more than once? We don't need a separate component if we're not reusing it.
But, this and the two buttons above it are nearly identical. Can we combine them and just pass the className as a prop? then we'll have reusable components